UnitedIncome/components
# Adds the Dollar Circle Illustration

### Description
> Please provide a brief description of your changes below.

Implements the `DollarCircleIllustration` to the component library.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Observe it in Storybook under `atoms/illustrations`.

### Comments
> Any additional comments that will aid with the review process.

Used for personal loan things.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
